kafka_consumer/datadog_checks/kafka_consumer/kafka_consumer.py

```diff
@@ -1,23 +1,36 @@
 # (C) Datadog, Inc. 2019-present
 # All rights reserved
 # Licensed under Simplified BSD License (see LICENSE)
+import json
+from collections import defaultdict
 from time import time
 
-from datadog_checks.base import AgentCheck
+from datadog_checks.base import AgentCheck, is_affirmative
 from datadog_checks.kafka_consumer.client import KafkaClient
 from datadog_checks.kafka_consumer.config import KafkaConfig
 
+try:
+    from datadog_agent import read_persistent_cache, write_persistent_cache
+except ImportError:
+    from datadog_checks.base.stubs import datadog_agent
+
+    read_persistent_cache = datadog_agent.read_persistent_cache
+    write_persistent_cache = datadog_agent.write_persistent_cache
```

Comment:
Is there a specific reason to not use [the methods](https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/datadog_checks/base/checks/base.py#L1013-L1039) provided in the base check?

Even if this is something that is done in other checks, it would be nice to avoid importing the stuff in the check